docs/aerodromes/classd/Moorabbin.md

```diff
@@ -81,13 +116,88 @@ Aircraft who will transit Class G airspace on climb into CTA will be **cleared t
     **SGE**: "Cleared to leave and re-enter controlled airspace climbing to A050, Departures 129.4, SGE"
 
 ### Arrivals/Overfliers
-ML TCU will heads-up coordinate arrivals/overfliers from Class C to MB ADC.  
-IFR aircraft will be cleared for the coordinated approach (Instrument or Visual) prior to handoff to MB ADC, unless MB ADC nominates a restriction.  
-VFR aircraft will not be coordinated, and will need to receive their airways clearance from MB ADC.
+
+- ML TCU will heads-up coordinate arrivals/overfliers from Class C to MB ADC.  
+- **IFR arrivals** will be cleared for the coordinated approach (Instrument or Visual) prior to handoff to MB ADC, unless MB ADC nominates a restriction.  
+- **VFR arrivals** will not be coordinated, and will need to receive their airways clearance from MB ADC.
 
 !!! example
     <span class="hotline">**MDS** -> **MB ADC**</span>: "To the west, PLE, for the NDB-A"  
     <span class="hotline">**MB ADC** -> **MDS**</span>: "PLE, NDB-A"
 
 !!! tip
-    Remember that IFR aircraft are only separated from other IFR or SVFR aircraft in class D. Use other separation methods, such as holding a departure on the ground, if separation is required with these aircraft.
\ No newline at end of file
+    Remember that IFR aircraft are only separated from other IFR or SVFR aircraft in class D. Use other separation methods, such as holding a departure on the ground, if separation is required with these aircraft.
+
+## Helicopters
```

Comment:
Would you mind moving this section above the Coordination section, for standardisation with other pages?